backend/src/controllers/UserController.js

```diff
@@ -0,0 +1,42 @@
+const connection = require('../database/connection');
+
+module.exports = {
+  async index (request, response) {
+    const user = await connection('user').select('*');
+    
+    return response.json(user);
+  },
+
+  async create (request, response) {
+    const { username, password, email, active } = request.body;
+
+    try { 
+      await connection('user').insert({
+        username,
+        password,
+        email,
+        active
+      });
+      } catch (exception) {
+        console.log('Exception: ' + exception);
+        return response.status(400).send();
+      }
+
+      return response.status(200).send();
```

Comment:
Suggestion: change the response's status code to 204 if the endpoint doesn't return any data.

Reply:
Done. And i added to the Delete method too.